Duplicati/Library/Backend/Tardigrade/TardigradeBackend.cs

```diff
@@ -44,6 +44,11 @@ public class Tardigrade : IStreamingBackend
             { "Access grant", "Access grant" },
         };
 
+        static Tardigrade()
+        {
+            Access.SetTempDirectory(System.IO.Path.GetTempPath());
```

Comment:
The temporary directory can be set by the user.  Should we reference `Library.Utility.TempFolder.SystemTempPath` instead?

Reply:
Ah, didn't know that one. Thanks @warwickmm - resolved it!